### PR TITLE
Ignore .mypy_cache folder

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ codespell.egg-info
 *.orig
 .cache/
 .pytest_cache/
+.mypy_cache/
```
Comment (peternewman) [inline .gitignore:10]: Sort the entries:
```suggestion
.cache/
.mypy_cache/
.pytest_cache/
```

**Review by peternewman — APPROVED** [review]
LGTM

I assume you're aware this is just for development and not for codespell itself?

**Review by peternewman — APPROVED** [review]
We don't use it currently, but there's no reason not to make life easier for those who do...

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**